content/en/docs/reference/access-authn-authz/service-accounts-admin.md

```diff
@@ -23,96 +23,108 @@ incomplete features are referred to in order to better describe service accounts
 Kubernetes distinguishes between the concept of a user account and a service account
 for a number of reasons:
 
-  - User accounts are for humans. Service accounts are for processes, which
-    run in pods.
-  - User accounts are intended to be global. Names must be unique across all
-    namespaces of a cluster, future user resource will not be namespaced.
-    Service accounts are namespaced.
-  - Typically, a cluster's User accounts might be synced from a corporate
-    database, where new user account creation requires special privileges and
-    is tied to complex business processes. Service account creation is intended
-    to be more lightweight, allowing cluster users to create service accounts for
-    specific tasks (i.e. principle of least privilege).
-  - Auditing considerations for humans and service accounts may differ.
-  - A config bundle for a complex system may include definition of various service
-    accounts for components of that system.  Because service accounts can be created
-    ad-hoc and have namespaced names, such config is portable.
+- User accounts are for humans. Service accounts are for processes, which run
+  in pods.
+- User accounts are intended to be global. Names must be unique across all
+  namespaces of a cluster. Service accounts are namespaced.
+- Typically, a cluster's user accounts might be synced from a corporate
+  database, where new user account creation requires special privileges and is
+  tied to complex business processes. Service account creation is intended to be
+  more lightweight, allowing cluster users to create service accounts for
+  specific tasks (i.e. principle of least privilege).
+- Auditing considerations for humans and service accounts may differ.
+- A config bundle for a complex system may include definition of various service
+  accounts for components of that system. Because service accounts can be created
+  ad-hoc and have namespaced names, such config is portable.
 
 ## Service account automation
 
 Three separate components cooperate to implement the automation around service accounts:
 
-  - A Service account admission controller
-  - A Token controller
-  - A Service account controller
+- A `ServiceAccount` admission controller
+- A Token controller
+- A ServiceAccount controller
```

Comment:
nit: Are you formatting lines 44 and 46?

Reply:
Yes.